app/views/shared/_accordion.html.erb

```diff
@@ -0,0 +1,15 @@
+<div class="usa-accordion usa-accordion--bordered">
+  <<%= options.fetch(:heading_level, 'div')%> class="usa-accordion__heading">
```

Comment:
WDYT about calling this `heading_tag` or `heading_elem`? The "level" name makes me think it would be like `h1` vs `h2`, etc.

Reply:
And also.... it looks like we never use anything beside `<div>` so should we just remove the option?

Reply:
> WDYT about calling this `heading_tag` or `heading_elem`? The "level" name makes me think it would be like `h1` vs `h2`, etc.

Not sure if it's what you meant, but I can see how there might be some confusion here in seeing the level as the numeric value (`1`, `2`, `3`), where as implemented currently it's actually the tag name (`'h1'`, `'h2'`, `'h3'`). The conclusion would be the same: `heading_tag` would probably be a better name 😄 

> And also.... it looks like we never use anything beside `<div>` so should we just remove the option?

You'd be right, but the way I was looking at it was: The fact that we don't use anything other than `div` is the error that should be fixed, where the ideal usage would provide a heading level, to (1) [align with Login.gov Design System code guidance](https://design.login.gov/components/accordions/) and (2) to provide semantic meaning as headings. This implementation optimizes for that future state by providing a path to achieve it.